Reg support radio test (#10749)
nrfconnect/sdk-nrf: samples/wifi/radio_test/sample_description.rst

              [00:24:25.202,606] <inf> otp_prog: Written REGION_DEFAULTS (0x154) : 0xfffffffb
              [00:24:25.203,002] <inf> otp_prog: Finished Writing OTP params

         * To set a regulatory domain with the following configuration:

           * Regulatory domain: US

           Execute the following command:

           .. code-block:: console

              wifi_radio_test reg_domain US

           The sample shows the following output:

           .. code-block:: console

				wifi_radio_test show_config

				reg_domain = US

         .. note::

>             * The default regulatory domain is 00 (World regulatory).

Reviewer: ```suggestion
            The default regulatory domain is 00 (World regulatory).
```

		 * To bypass regulatory domain, set ``bypass_reg_domain`` to ``1`` using the following command:

> 
> 			* bypass_reg_domain: 1

Reviewer: ```suggestion

```

>            Execute the following sequence of command:

Reviewer: ```suggestion
    
```

> 

Reviewer: ```suggestion
```

Author: This is probably needed.  (In line 590 a blank line is needed)

           .. code-block:: console

              wifi_radio_test bypass_reg_domain 1

           The sample shows the following output:

           .. code-block:: console

               wifi_radio_test show_config
               reg_domain = US

>                bypass_reg_domain = 1

Reviewer: Lines 585-601 need not to be in Note.

         .. note::

>             * Bypass regulatory domain is false by default.

Reviewer: ```suggestion
            Bypass regulatory domain is false by default.
```

>             See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands.

Reviewer: Line 607: It should be out of the Note.

              [00:24:25.202,606] <inf> otp_prog: Written REGION_DEFAULTS (0x154) : 0xfffffffb
              [00:24:25.203,002] <inf> otp_prog: Finished Writing OTP params

         * To set a regulatory domain with the following configuration:

           * Regulatory domain: US

           Execute the following command:

           .. code-block:: console

              wifi_radio_test reg_domain US

           The sample shows the following output:

           .. code-block:: console

              wifi_radio_test show_config
              reg_domain = US

         .. note::

            The default regulatory domain is ``00`` (World regulatory).

>          * To bypass regulatory domain, set ``bypass_reg_domain`` to ``1`` using the following command:

Reviewer 2: Exact meaning of `byapss` needs some doc, e.g., which rules it follow? or just accepts any channel or power etc?

Reviewer 3: updated with description

              [00:24:25.202,606] <inf> otp_prog: Written REGION_DEFAULTS (0x154) : 0xfffffffb
              [00:24:25.203,002] <inf> otp_prog: Finished Writing OTP params

         * To set a regulatory domain with the following configuration:

           * Regulatory domain: US

           Execute the following command:

           .. code-block:: console

              wifi_radio_test reg_domain US

           The sample shows the following output:

           .. code-block:: console

              wifi_radio_test show_config
              reg_domain = US

         .. note::

            The default regulatory domain is ``00`` (World regulatory).

         * To bypass regulatory domain, set ``bypass_reg_domain`` to ``1`` using the following command:

           .. code-block:: console

              wifi_radio_test bypass_reg_domain 1

           The sample shows the following output:

           .. code-block:: console

               wifi_radio_test show_config
               reg_domain = US
               bypass_reg_domain = 1

         .. note::

            Bypass regulatory domain is false by default.

>             If 'bypass_reg_domain = 0', then TX power of the channel will be configured to the minimum value of
>             the user configured TX power value and max power supported in the configured regulatory domain.

Reviewer: ```suggestion
            If ``bypass_reg_domain`` is ``0``, then TX power of the channel will be configured to the minimum value of the user configured TX power value and maximum power supported in the configured regulatory domain.
```

>             If 'bypass_reg_domain = 1', then user configured TX power value will be set overriding current
>             configured regulatory domain max TX power for the channel.

Reviewer: ```suggestion
            If ``bypass_reg_domain`` is ``1``, then user configured TX power value will be set overriding current configured regulatory domain maximum TX power for the channel.
```

              [00:24:25.203,002] <inf> otp_prog: Finished Writing OTP params

         See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands.
         
         * To set regulatory domain with the following configuration:

>            * regulatoy domain: US

Reviewer 4: regulatory spelling

         See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands.

>          * To set regulatory domain with the following configuration:

Reviewer: Line 562 "See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands." can be moved after line 605.

           * regulatory domain: US

           Execute the following sequence of commands:

           .. code-block:: console

              wifi_radio_test reg_domain US

           The sample shows the following output:

           .. code-block:: console

               wifi_radio_test show_config
               reg_domain = US

         .. note::

            The default regulatory domain is 00(World regulatory).

>          * To bypass regulatory domain set use the following configuration:
> 
>            * bypass_reg_domain: 1

Reviewer: ```suggestion
         * To bypass regulatory domain, set ``bypass_reg_domain`` to ``1`` using the following command:
```

> 
>            Execute the following sequence of commands:

Reviewer: ```suggestion
```
